desc/objectives/getters.py

```diff
@@ -191,14 +195,84 @@ def get_NAE_constraints(
         FixAxisZ(eq=desc_eq, normalize=normalize, normalize_target=normalize),
         FixPsi(eq=desc_eq, normalize=normalize, normalize_target=normalize),
     )
-
     if profiles:
         for name, con in _PROFILE_CONSTRAINTS.items():
             if getattr(desc_eq, name) is not None:
                 constraints += (
                     con(eq=desc_eq, normalize=normalize, normalize_target=normalize),
                 )
 
+    constraints += (
+        FixNearAxisR(
+            eq=desc_eq,
+            nae_eq=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+        FixNearAxisZ(
+            eq=desc_eq,
+            nae_eq=qsc_eq,
+            N=N,
+            order=order,
+            normalize=normalize,
+        ),
+    )
+    if fix_lambda or (fix_lambda >= 0 and type(fix_lambda) is int):
+        constraints += (
+            FixNearAxisLambda(
+                eq=desc_eq,
+                nae_eq=qsc_eq,
+                N=N,
+                order=int(fix_lambda),
+                normalize=normalize,
+            ),
+        )
+
+    return constraints
+
+
+def _get_NAE_constraints(
```

Comment:
Why do we still have this function? I think this can be removed

Reply:
I use it to create the constraints in `FixNearAxisXXX`